aiosmtpd/tests/conftest.py

```diff
@@ -315,27 +321,52 @@ def client(request: pytest.FixtureRequest) -> Generator[SMTPClient, None, None]:
 
 
 @pytest.fixture
-def ssl_context_server() -> ssl.SSLContext:
+def tls_certificate_authority() -> trustme.CA:
+    if not TRUSTME:
+        pytest.xfail("trustme is not supported")
+    return trustme.CA()
+
+
+@pytest.fixture
+def tls_certificate(tls_certificate_authority: trustme.CA) -> trustme.LeafCert:
+    return tls_certificate_authority.issue_cert(
+        "localhost",
+        "xn--prklad-4va.localhost",
```

Comment:
Let's log the decoded value here
```suggestion
        "xn--prklad-4va.localhost",  # príklad.localhost
```